client/components/RatingsReviews/ReviewList.jsx

```diff
@@ -14,19 +15,31 @@ const getReviews = (pId) => {
 export const ReviewList = (props) => {
   const [loading, setLoading] = useState(true);
```

Comment:
use of loading state variable seems unnecessary. although this code is fine, maybe consider adding a small tech debt ticket to refactor to useEffect hooks? These accomplish similar things to class lifecycle methods.